mountinfo/mountinfo_freebsdlike.go

```diff
@@ -0,0 +1,13 @@
+//go:build freebsd || darwin
+// +build freebsd darwin
+
+package mountinfo
+
+import "syscall"
+
+func getMountinfo(entry syscall.Statfs_t) (mountinfo Info) {
+	mountinfo.Mountpoint = int8SliceToString(entry.Mntonname[:])
```

Comment:
This smells like double data copying to me

Reply:
I changed `func getMountinfo(entry syscall.Statfs_t) Info` to `func getMountinfo(entry *unix.Statfs_t) *Info`.